ql/src/test/results/clientpositive/llap/auto_join12.q.out

```diff
@@ -128,6 +108,16 @@ STAGE PLANS:
                         Map-reduce partition columns: _col0 (type: string)
                         Statistics: Num rows: 166 Data size: 29548 Basic stats: COMPLETE Column stats: COMPLETE
                         value expressions: _col1 (type: string)
+                    Select Operator
+                      expressions: key (type: string)
+                      outputColumnNames: _col0
+                      Statistics: Num rows: 166 Data size: 14442 Basic stats: COMPLETE Column stats: COMPLETE
+                      Reduce Output Operator
+                        key expressions: _col0 (type: string)
```

Comment:
Just a note (no action needed). It seems here we could have reused the same shuffle if it wasn't for the additional value column in one of them?

Reply:
yes ..even if the tree is doing the same the rs is retained ;  we could probably improve on this later on...